README.md

```diff
@@ -1,39 +1,26 @@
-[![Coverage Status](https://github.com/ros-controls/ros2_control_demos/workflows/Coverage/badge.svg?branch=master)](https://github.com/ros-controls/ros2_control_demos/actions?query=workflow%3ACoverage)
-[![Licence](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+## Introduction
 
-This repository provides templates for the development of `ros2_control`-enabled robots and a simple simulations to demonstrate and prove `ros2_control` concepts.
+`ros2_control_bolt` is a repository that enables designing and simulation of Bolt robot for testing its controller.
 
-## Goals
 
-The repository has three goals:
-1. Implements the example configuration described in the `ros-controls/roadmap` repository file [components_architecture_and_urdf_examples](https://github.com/ros-controls/roadmap/blob/master/design_drafts/components_architecture_and_urdf_examples.md).
-2. It provides templates for faster implementation of custom hardware and controllers;
-3. The repository is a validation environment for `ros2_control` concepts, which can only be tested during run-time (e.g., execution of controllers by the controller manager, communication between robot hardware and controllers).
+## Repository Organization
 
+`doc` - a set of tutorials to start, calibrate and launch Bolt.
 
-## Description
+`gazebo_ros2_control_bolt` - a ROS2 package that allows to simulate Bolt robot on Gazebo and apply position, velocity, effort and gains commands to it.
 
-The repository is inspired by the [ros_control_boilerplate](https://github.com/PickNikRobotics/ros_control_boilerplate) repository from Dave Coleman.
-The simulation has three parts/packages:
-1. The first package, `ros2_control_demo_bringup`, holds launch files and runtime configurations for demo robots.
-2. The second package, `ros2_control_demo_description`, stored URDF-description files, rviz configurations and meshes for the demo robots.
-3. The third package, `ros2_control_demo_hardware`, implements the hardware interfaces described in the roadmap.
-The examples simulate a simple *RRbot* internally to provide sufficient test and demonstration data and reduce external dependencies.
-This package does not have any dependencies except `ros2` core packages and can, therefore, be used on SoC-hardware of headless systems.
+`position_velocity_effort_gain_controller` - a ROS2 package in which is implemented a simple controller that enables to send position, velocity, effort and gain commands to the robot, with the specific hardware interface.
 
-This repository demonstrates the following `ros2_control` concepts:
+`ros2_control_bolt_bringup` - contains files that enable to launch Bolt within its GUIs, and with it hardware interfaces and controllers.
+
+`ros2_control_test_nodes` - contains nodes for testing ros2_control-enabled robots and their integration in the framework.
```

Comment:
Maybe this package should be removed if this is the same than the initial `ros2_control_demos`repository.